tldw_Server_API/tests/sandbox/test_artifact_traversal_integration.py

```
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import os
+import threading
+import time
+from typing import Any
+
+import pytest
+
+
+@pytest.mark.integration
+def test_artifact_traversal_rejected_under_uvicorn(tmp_path) -> None:
```

Ruff (0.14.2)

Unused function argument: `tmp_path`

(ARG001)